plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/ConvertJobSuspenderTest.java

```diff
@@ -41,7 +40,7 @@
   @Before
   public void setUp() {
     // Assume all test jobs are "ConvertJob"s.
-    ConvertJobSuspender.isConvertJob = Predicates.alwaysTrue();
+    ConvertJobSuspender.isConvertJob = job -> true;
```

Comment:
Aside: I personally find Guava's `alwaysTrue()` and `alwaysFalse()` to be clearer — I don't have to parse the actual lambda and see what it's doing.  That said, until we update to a later Guava, the `Predicate`s and `Function`s aren't interchangeable.

Reply:
We should update to a later Guava, but we should probably start further down the stack in appengine-plugins-core and ide-login.